theme/themes/pastanaga/collections/table.variables

```diff
@@ -18,7 +18,7 @@
 @headerBackground: #FAF8F8;
 @headerFontWeight: @normal;
 @headerTextTransform: uppercase;
-@headerColor: #777776;
+@headerColor: #878f93;
```

Comment:
That one will apply to all tables, we have to make sure we want that.

Reply:
How I can implement a particular CSS for this table and applied to it. This is the only part I did not understand until now?